src/main/java/com/izmaylov/hw10/carModel/Car.java

```diff
@@ -0,0 +1,67 @@
+package com.izmaylov.hw10.carModel;
+
+public abstract class Car {
```

Comment:
Put `implements Recovery` here and you will not need to specify in all child classes